src/python/SignalStats.i

```diff
@@ -0,0 +1,107 @@
+%}
+
+%include "std_string.i"
+%include "std_map.i"
+%template(map_string_double) std::map<std::string, double>;
+
+namespace ignition
+{
+  namespace math
+  {
+    class SignalMaximum
+    {      
+      %rename("%(undercase)s", %$isfunction, %$ismember, %$not %$isconstructor) "";
+      public: virtual void Reset();
+      public: virtual size_t Count() const;
+      public: virtual double Value() const;
+      public: virtual std::string ShortName() const;
+      public: virtual void InsertData(const double _data);
+    };
+
+    class SignalMean
+    {      
+      %rename("%(undercase)s", %$isfunction, %$ismember, %$not %$isconstructor) "";
+      public: virtual void Reset();
+      public: virtual size_t Count() const;
+      public: virtual double Value() const;
+      public: virtual std::string ShortName() const;
+      public: virtual void InsertData(const double _data);
+    };
+
+    class SignalMinimum
+    {      
+      %rename("%(undercase)s", %$isfunction, %$ismember, %$not %$isconstructor) "";
+      public: virtual void Reset();
+      public: virtual size_t Count() const;
+      public: virtual double Value() const;
+      public: virtual std::string ShortName() const;
+      public: virtual void InsertData(const double _data);
+    };
+
+    class SignalRootMeanSquare
+    {      
+      %rename("%(undercase)s", %$isfunction, %$ismember, %$not %$isconstructor) "";
+      public: virtual void Reset();
+      public: virtual size_t Count() const;
+      public: virtual double Value() const;
+      public: virtual std::string ShortName() const;
+      public: virtual void InsertData(const double _data);
+    };
+
+    class SignalMaxAbsoluteValue
+    {      
+      %rename("%(undercase)s", %$isfunction, %$ismember, %$not %$isconstructor) "";
+      public: virtual void Reset();
+      public: virtual size_t Count() const;
+      public: virtual double Value() const;
+      public: virtual std::string ShortName() const;
+      public: virtual void InsertData(const double _data);
+    };
+
+    class SignalVariance
+    {      
+      %rename("%(undercase)s", %$isfunction, %$ismember, %$not %$isconstructor) "";
+      public: virtual void Reset();
+      public: virtual size_t Count() const;
+      public: virtual double Value() const;
+      public: virtual std::string ShortName() const;
+      public: virtual void InsertData(const double _data);
+    };
+
+    class SignalStats
+    {      
+      %rename("%(undercase)s", %$isfunction, %$ismember, %$not %$isconstructor) "";
+      public: SignalStats();
+      public: ~SignalStats();
+      public: SignalStats(const SignalStats &_ss);
+      public: size_t Count() const;
+      public: std::map<std::string, double> Map() const;
+      public: void InsertData(const double _data);
+      public: bool InsertStatistic(const std::string &_name);
+      public: bool InsertStatistics(const std::string &_names);
+      public: void Reset();
+    };
+
+  }
+}
```

Comment:
nit: Add blank line at the end of the file

Reply:
Addressed.